samples/scenarios/vendor_attributes/postgres_vendor/dialect.tdd

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
+<dialect name='SimplePostgres'
+         class='postgres_vendor'
+         base='PostgreSQL90Dialect'
+         version='18.1'>
+   <function-map>
```

Comment:
For the scenario based ones we can just use the base dialect as is.  This FLOAT was provided as an example originally.